unfoldingWord/translationCore
# Missing Verses Check

#### This pull request addresses:
This PR looks for missing verses in the selected project and displays them to the user.

#### How to test this pull request:
Try opening a project and seeing if the missing verses are displayed at the corresponding check step. Projects with no missing verses should not show the step


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2445)
